### ISSUE TITLE
[Bug]: mysql dependency breaks after new deno_std release 0.160.1

### ISSUE THREAD

**Issue by mechanoid** [halvardssm/deno-nessie#160]
### Operating System

macOS Ventura 13.0

### Deno version

1.26.2

### Nessie version

2.0.9

### Bug description

deno_std@0.160.1 was released today, and the mysql driver depends on god_crypto, that had incorporated the std lib without version reference. The mysql driver has an issue for that and already implemented a fix (see https://github.com/denodrivers/mysql/issues/131) which is not yet released as far as I can see. 

This results in an error like:

```
Warning Implicitly using latest version (0.161.0) for https://deno.land/std/hash/mod.ts
error: Module not found "https://deno.land/std/hash/mod.ts".
    at https://deno.land/x/god_crypto@v0.2.0/src/eme_oaep.ts:1:28
```

Would be great if nessie can be updated short after the mysql patch has been delivered. 

Thanks a lot in advance!

### Steps to reproduce

1. install nessie 
2. run nessie

**Comment by mechanoid** [comment]
thx @pandres95 !

**Comment by halvardssm** [comment]
Hey @pandres95 @mechanoid ! New version has been released `2.0.10` 🎉

**Comment by mechanoid** [comment]
@halvardssm super cool, thx a lot 🙇‍♂️
